src/components/Resources/useFilters.ts

```diff
@@ -14,18 +14,18 @@ interface UseFilters {
     resetFilterGroups: () => void
 }
 
+interface UseFilterProps {
+    telemetryRecorder: TelemetryRecorder<'', ''>
+}
+
 /**
  * This hook is used for getting, setting, and resetting filters
  * based on our resource items data.
  */
-export const useFilters = (): UseFilters => {
-    const logResourceClickEvent = (groupTitle: string, type: string): void => {
-        const eventName =
-            groupTitle === 'Content Type' ? EventName.RESOURCE_CONTENT_TYPE_FILTER : EventName.RESOURCE_SUBJECT_FILTER
+export const useFilters = ({ telemetryRecorder }: UseFilterProps): UseFilters => {
+    const logResourceClickEvent = (groupTitle: string, type: string): void =>
+        telemetryRecorder.recordEvent(`resources.filter.${groupTitle === 'Content Type' ? 'contentType' : 'subject'}`, 'toggle', { privateMetadata: { type }})
```

Comment:
I see that previously, we used different event names, and that's why we do it in the new telemetry, but wouldn't it be better if we used one event name but put this content type property in the event metadata?